### PR TITLE
[KYUUBI #5447][FOLLOWUP] Remove unrelated debug prints in TableIdentifierTableExtractor

### LINKED ISSUE #5447: [Subtask] Support Hudi Delete/Update/MergeInto Table Command
### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### Search before asking

- [X] I have searched in the [issues](https://github.com/apache/kyuubi/issues) and found no similar issues.

### Describe the subtask

 Support Hoodie Delete/Update/MergeInto Table Command

### Parent issue

#5285 

### Are you willing to submit PR?

- [X] Yes. I would be willing to submit a PR with guidance from the Kyuubi community to improve.
- [ ] No. I cannot submit a PR at this time.

### PR DESCRIPTION
**Pull request by AngersZhuuuu**
### _Why are the changes needed?_
To close #5447 
Remove unrelated debug change in  tableExtractor

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] [Run test](https://kyuubi.readthedocs.io/en/master/contributing/code/testing.html#running-tests) locally before make a pull request

### _Was this patch authored or co-authored using generative AI tooling?_
No

### CODE REVIEW

**Comment by AngersZhuuuu** [comment]
ping @bowenliang123

File: extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/serde/tableExtractors.scala [hunk 1]
```diff
@@ -80,9 +80,7 @@ class TableIdentifierTableExtractor extends TableExtractor {
         val catalogTable = spark.sessionState.catalog.getTableMetadata(identifier)
         Option(catalogTable.owner).filter(_.nonEmpty)
       } catch {
-        case e: Exception =>
-          e.printStackTrace()
-          None
+        case e: Exception => None
```
Comment (bowenliang123) [inline extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/serde/tableExtractors.scala:83]:
```suggestion
        case _: Exception => None
```

**Review by bowenliang123 — APPROVED** [review]
